examples/your-standard-node-app.inlineplz.yml

```diff
@@ -0,0 +1,24 @@
+ignore_paths:
+  - node_modules
+  - vendors
+  - .tox
+  - .git
+enabled_linters:
+  - markdown
+  - eslint
```

Comment:
I want to remove `lint`, but it makes no sense to have `es`...

Reply:
This gets a bit at what I'm thinking, which is that some of these linter names are actually proper nouns and when there's more than one available / widely known for a format it may be nice to know which concrete linter is getting used (for rules / config etc.)

Reply:
Ed! You are alive! I tried to reach out but you are a ghost 👻 So, I sent you a message to your work email.

---

Yeah, had the same thought, but still wanted to see where this lands.